core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -54,12 +51,47 @@ describe('Topics and skills dashboard functionality', function() {
     await topicsAndSkillsDashboardPage.get();
   });
 
-  it('should add a new topic to list', async function() {
+  it('should add a new topic to list and delete it', async function() {
     await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
-    await topicsAndSkillsDashboardPage.createTopic('Topic 1', true);
+    await topicsAndSkillsDashboardPage.createTopic(
+      'Topic 1', 'Topic 1 description', 'Mathematics', true);
 
     await topicsAndSkillsDashboardPage.get();
     await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(1);
+    await topicsAndSkillsDashboardPage.deleteTopicWithIndex(0);
+    await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
+  });
+
+  it('should filter the topics', async function() {
+    await topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
+    await topicsAndSkillsDashboardPage.createTopic(
+      'Alpha Topic Dashboard', 'Alpha description', 'Mathematics', true);
```

Comment:
Per the conventions on the wiki, can we include either `topicsAndSkillsDashboard` or `TASD` to ensure uniqueness?

Reply:
Done.